test/end-to-end/pages/iht/method.js

```diff
@@ -1,17 +1,23 @@
 'use strict';
 
 const config = require('config');
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/iht/method');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
+const contentEn = require('app/resources/en/translation/iht/method');
+const contentCy = require('app/resources/cy/translation/iht/method');
 
-module.exports = async function(answer) {
+module.exports = async function(language = 'en', answer) {
     const I = this;
+    const commonContent = language === 'en' ? commonContentEn : commonContentCy;
+    const ihtContent = language === 'en' ? contentEn : contentCy;
 
     await I.checkPageUrl('app/steps/ui/iht/method');
-    await I.waitForText(content.question, config.TestWaitForTextToAppear);
+    if (language ==='en') {
+        await I.waitForText(ihtContent.question, config.TestWaitForTextToAppear);
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages